backend/qemu.pm

```diff
@@ -622,6 +624,13 @@ sub start_qemu {
     $self->{proc}->qemu_bin($qemubin);
     $self->{proc}->qemu_img_bin($qemuimg);
 
+    # Get qemu version
+    my $qemu_version = `$qemubin -version`;
+    $qemu_version =~ /([0-9]+([.][0-9]+)+)/;
+    $qemu_version = $1;
+    $vars->{QEMU_VERSION} = $qemu_version;
```

Comment:
I know that `bmwqemu::vars` are apparently the place to store everything in this project but wouldn't it make more sense to store the version within the backend object, e.g. `$self->{qemu_version}`?

Reply:
+1

looks like an abuse of test variables for persistent storage that IMHO does not even need persistent storage